modules/bitgo/test/v2/unit/coins/abstractUtxoCoin.ts

```diff
@@ -252,7 +260,10 @@ describe('Abstract UTXO Coin:', () => {
         badKey: {
           keychains: {
             // user public key swapped out
-            user: { pub: otherKeychain.pub, encryptedPrv: bitgo.encrypt({ input: userKeychain.prv, password: passphrase }) },
+            user: {
+              pub: otherKeychain.pub,
+              encryptedPrv: bitgo.encrypt({ input: userKeychain.prv, password: passphrase }),
+            },
```

Comment:
it looks like we have dueling prettier configs or something

```
git ls-files | grep prettierrc
```

at the toplevel paints a scary picture

I guess your IDE picked a different prettierrc that the the CI that checks the formatting? 

Either bring the prettier configs into harmony and submit a reformat PR first or unstange the formatting changes from this PR

Reply:
https://github.com/BitGo/BitGoJS/pull/3698